pkg/resources/validation/feature/feature.go

```diff
@@ -49,6 +49,18 @@ func (v *Validator) Admit(request *admission.Request) (*admissionv1.AdmissionRes
 		return nil, err
 	}
 
+	if oldFeature.Name == "multi-cluster-management" && !isValidMCMFeatureValue(oldFeature.Status.Default, oldFeature.Spec.Value, newFeature.Spec.Value) {
```

Comment:
Is there a constant definition of this flag?

Reply:
I can only find this line in Rancher repo. Other files are YAML.

https://github.com/rancher/rancher/blob/dc282bc553b17277c5e7ae1b871857f049842445/pkg/features/feature.go#L33